Makefile.js

```diff
@@ -327,6 +328,44 @@ function hasBranch(branchName) {
     return branches.indexOf(branchName) !== -1;
 }
 
+
+/**
+ * Generates a release blog post for eslint.org
+ * @param {Object} releaseInfo The release metadata.
+ * @returns {void}
+ * @private
+ */
+function generateBlogPost(releaseInfo) {
+    var output = ejs.render(cat("./templates/blogpost.md.ejs"), releaseInfo),
+        now = new Date(),
+        month = now.getMonth() + 1,
+        day = now.getDate(),
+        filename = "../eslint.github.io/_posts/" + now.getFullYear() + "-" +
+            (month < 10 ? "0" + month : month) + "-" +
+            (day < 10 ? "0" + day : day) + "-eslint-" + releaseInfo.version +
+            "-released.md";
+
+    output.to(filename);
+}
+
+/**
+ * Given a semver version, determine the type of version.
+ * @param {string} version A semver version string.
+ * @returns {string} The type of version.
+ * @private
+ */
+function getReleaseType(version) {
+    var parts = version.split(".");
```

Comment:
Would it be more robust to use [node-semver](https://github.com/npm/node-semver) and it's `major(version)`, `minor(version)`, `patch(version)` functions?  Although I can't think of any cases where splitting like this would cause problems.